tests/panels/charts/xy/test_xy.py

```
@@ -333,3 +333,231 @@ async def test_area_percentage_chart() -> None:
+            {
+                'label': 'Target',
+                'value': 200.0,
+                'axis': 'left',
+                'color': '#00FF00',
+                'line_style': 'solid',
+            },
+        ],
+    }
+
+    lens_chart = LensLineChart(**lens_config)
+    layer_id, kbn_columns, kbn_state_visualization = compile_lens_xy_chart(lens_xy_chart=lens_chart)
```

Ruff (0.14.10)

Unpacked variable `layer_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)